ROADMAP.md

```diff
@@ -0,0 +1,55 @@
+# cellxgene roadmap
```

Comment:
Suggest the first para needs to give a pointer to where/how we would like input (eg, "if you have roadmap feedback, please submit as a github issue")